uaclient/api/api.py

```diff
@@ -0,0 +1,55 @@
+from importlib import import_module
+from typing import Callable, Optional
+
+from uaclient.api.data_types import APIData, APIResponse
+from uaclient.api.utils import VALID_ENDPOINTS, error_out
+from uaclient.data_types import DataObject
+
+
+def call_api(args, *, cfg):
```

Comment:
I think either this class should accept args that aren't tied to the cli argparser implementation, or the function should be moved to the cli module.

Reply:
Added a wrapper, better indeed.